src/api.raml

```diff
@@ -185,6 +185,28 @@ traits:
               required: false
               example: [article/1234, interview/5678]
 
+/annotation-token/{id}:
+    type:
+        item:
+            resource: Annotation token
+    uriParameters:
+        id:
+            description: |
+                Annotation author ID.
+            type: string
+    get:
+        description: |
+            Get an annotation token.
+        responses:
+            200:
+                description: |
+                    Annotation token.
+                body:
+                    application/vnd.elife.annotation-token+json;version=1:
```

Comment:
I wonder if there is a standard way to mark this as experimental as we will need to merge to go live, and do breaking changes later. Perhaps there is a standard field in RAML? Or could we use `version=0`?

Reply:
Let's discuss this with @thewilkybarkid.

Reply:
RAML has annotations. Thinking about it, we should do that for deprecations too. I'll take a look at adding them.

Reply:
We've started using them, so the whole API should be `@experimental`.